application/controllers/admin/ParticipantsAction.php

```diff
@@ -2373,7 +2374,7 @@ public function rejectShareParticipant()
      * @param int $shareUid
      * @return void
      */
-    public function deleteSingleParticipantShare($participantId, $shareUid)
+    public function deleteSingleParticipantShare($participantId = null, $shareUid = null)
```

Comment:
i think this might be a mistake, those 2 funtion params are needed. They are missing in the calling function, theres also a comment for it.

Reply:
Maybe yeah. Question is, what's the proper fix? :scream:

Reply:
Wanna move this out to a separate bug?

Reply:
Added to https://bugs.limesurvey.org/view.php?id=18850

Reply:
> Maybe yeah. Question is, what's the proper fix? scream

Move participantId to POST value not param.

Reply:
Yeah, good point @Shnoulle